other/web-js/client/src/components/WelcomePage.js

```diff
@@ -77,6 +122,7 @@ export function WelcomePage() {
           <NonAuthErrorAlert />
           <TextField
             id="input-email"
+            data-testid="input-email"
```

Comment:
should these `data-testid` props be removed? applies to all others as well

Reply:
Actually can remove this one but only because it's unused in the tests